### PR TITLE
[EDU-4818] feat: Update home texts and cards

### PR DESCRIPTION
**Pull request by mrmessa**
Update sections and cards to prioritize user intention

### CODE REVIEW

File: src/content/docs/en/homes/home-docs.mdx [hunk 1]
```diff
@@ -3,31 +3,107 @@ type: homepage
...
+        description: >-
+          View all use cases
+        link: >-
+          /en/documentation/architectures/
+  - title: Pick your development interface
+    cards:
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Azion Console
+        description: >- 
+          Manage and monitor your edgeapplications, services, and resources efficiently.
```
Comment (gabriel-azion) [inline src/content/docs/en/homes/home-docs.mdx:79]:
```suggestion
          Manage and monitor your edge applications, services, and resources efficiently.
```

**Review by gabriel-azion — APPROVED** [review]
just a typo, other than that lgtm

File: src/content/docs/en/homes/home-docs.mdx [hunk 2]
```diff
@@ -3,31 +3,107 @@ type: homepage
...
+          /en/documentation/architecture/edge-firewall/web-application-and-api-protection-waap/
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Serverless Applications
+        description: >-
+          Explore Azion Edge Functions for serverless applications, enhancing
+          scalability, performance, and real-time responsiveness at the edge.
+        link: >-
+          /en/documentation/architectures/edge-application/serverless-applications/
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Find more ...
```
Comment (MarianaAguilera) [inline src/content/docs/en/homes/home-docs.mdx:69]:
```suggestion
        title: Find more
```

File: src/content/docs/en/homes/home-docs.mdx [hunk 3]
```diff
@@ -3,31 +3,107 @@ type: homepage
...
+        description: >-
+          View all use cases
```
Comment (MarianaAguilera) [inline src/content/docs/en/homes/home-docs.mdx:71]:
```suggestion
          View all use cases.
```

File: src/content/docs/en/homes/home-docs.mdx [hunk 4]
```diff
@@ -3,62 +3,140 @@ type: homepage
...
       - icon: /assets/docs/images/uploads/icon-guides.svg
-        title: Guides
-        description: Practical documentation on how to use products and API.
-        link: /en/documentation/products/guides/
+        title: React
+        description: 'Build reusable UI components. Create dynamic web apps for edge infrastructure deployment.'
+        link: 'https://console.azion.com/create/react/react-boilerplate'
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Next
+        description: 'Create React apps with automatic routing and optimization for edge deployment.'
```
Comment (MarianaAguilera) [inline src/content/docs/en/homes/home-docs.mdx:20]:
```suggestion
        description: 'Create Next apps with automatic routing and optimization for edge deployment.'
```

File: src/content/docs/en/homes/home-docs.mdx [hunk 5]
```diff
@@ -3,62 +3,140 @@ type: homepage
...
+        title: Angular
+        description: 'Develop robust, scalable web applications using TypeScript for rapid edge deployment.'
+        link: 'https://console.azion.com/create/angular/angular-boilerplate'
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Svelte
+        description: 'Develop reactive UIs with less code, compiled for high-performance edge deployment.'
+        link: 'https://console.azion.com/create/svelte/svelte-boilerplate'  
+      - icon: /assets/docs/images/uploads/icon-guides.svg
+        title: Gatsby
+        description: 'Build fast, SEO-optimized sites using React and GraphQL for efficient edge delivery.'
```
Comment (MarianaAguilera) [inline src/content/docs/en/homes/home-docs.mdx:36]:
```suggestion
        description: 'Build fast, SEO-optimized sites using Gatsby for efficient edge delivery.'
```

File: src/content/docs/en/homes/home-docs.mdx [hunk 6]
```diff
@@ -3,62 +3,140 @@ type: homepage
...
+        title: Observe
         description: >-
-          Discover a range of solutions you can use to enhance, compose, or
-          customize your applications.
-        link: /en/documentation/products/marketplace/
-      - icon: /assets/docs/images/uploads/icon-account.svg
-        title: Accounts and Profile
+          Monitor and gain analytics in real-time.
+        link: /en/documentation/products/observe/overview/
+  - title: 'What is new?'
```
Comment (MarianaAguilera) [inline src/content/docs/en/homes/home-docs.mdx:129]:
```suggestion
  - title: 'What's new?'
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**